index.mjs

```diff
@@ -235,18 +237,28 @@ class MacCertStore extends CertStore {
 
 	static async _install(arg) {
 		await this.createTempFileIfNeeded(arg)
-		await exec(`security add-trusted-cert -d -r trustRoot -k "${MAC_DIR}" "${arg.path}"`)
+		const certPath = path.join(__dirname, arg.tempPath)
```

Comment:
have you tried looking at the method `createTempFileIfNeeded()` and the way `tempPath` is used https://github.com/MikeKovarik/cert-store/blob/master/index.mjs#L188 ?

Reply:
yes.`createTempFileIfNeeded` will not create a `.path`.  i updated this in another PR.